README.md

```diff
@@ -3,13 +3,29 @@
 - List 3 reasons why asymptotic analysis may be misleading with respect to
   actual performance in practice.
 
+  Asymptotics drops constants and lower order terms because asymptotics is focused on the big picture. As n gets larger and larger, lower order terms and exponents matter less, so when using asymptotics, we only need to care about the high order terms even though lower order terms have an impact on actual runtime with smaller values of n.
+  Asymptotics only gives general bounds for runtimes, as asymptotics provides answers as n gets larger and larger, meaning that could be a large difference in runtimes between two programs who both have the same asymptotic expression
```

Comment:
Don't understand what you mean here -- what does $n_0$ have to do with the difference between two algorithms?